linear_algebra/code/include/linear_algebra/debug_helpers.hpp

```diff
@@ -138,8 +138,8 @@ clean_type_name(basic_string<C,T,A> tname)
 inline void
 PrintFuncName(char const* name)
 {
-    cout << "********************************************************************************\n";
-    cout << "********************************************************************************\n";
+    cout << "--------------------------------------------------------------------------------\n";
```

Comment:
Something about this formatting was breaking the CMake scripts that automatically detect tests cases for registration with CTest.  Let me know if changing this is a problem, but I wanted to get individual test information displaying on the continuous integration.